hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,23 @@
+RELEASE_TYPE: patch
+
+This release improves the shrinker's ability to handle situations where there
+is an additive constraint between two values.
+
+For example, consider the following test:
+
+
+.. code-block:: python
+
+    import hypothesis.strategies as st
+    from hypothesis import given
+
+    @given(st.integers(), st.integers())
+    def test_does_not_exceed_100(m, n):
+        assert m + n <= 100
```

Comment:
To fail with `m=0, n=100`, this must use `<` not `<=`

Reply:
Whoops, yes, thanks.